docs/notebooks/neural_dual.ipynb

```diff
@@ -12,7 +12,7 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def get_optimizer(optimizer, lr, b1, b2, eps):
</pre><p>I don't think <code>get_optimizer</code> is  currently used, would remove and rename <code>get_optimizer_decay</code> to <code>get_optimizer</code> with <code>AdamW</code> as the only option (+ add defaults for <code>b1/b2</code>).    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/219/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
I'm still tweaking the optimizer around and will keep both for now. Once it's settled I'm also thinking about just inlining the optimizer

Reply:
Inlined